pom.xml

```diff
@@ -47,9 +47,9 @@
     <junit.version>4.12</junit.version>
     <slf4j.version>1.7.26</slf4j.version>
     <shaded.guava.version>28.2-jre</shaded.guava.version>
-    <guava.version>19.0</guava.version>
+    <guava.version>19.0</guava.version> <!--todo: 28.2-jre guava can be used here-->
```

Comment:
Let's create a ticket instead of adding the comment. Also, there are newer versions of Guava.

Reply:
ok. Yes, I know about newer version. But the newer version can bring some new other issues for sure.
The plan to check `<guava.version>28.2-jre</guava.version>` at first. If it is fine then to drop `<shaded.guava.version>` at all. And then to update guava version to the newest one

Reply:
So the comment can be removed?